Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst

```diff
@@ -66,6 +66,19 @@ class ClassName
     ) {
         return;
     }
+
+    public void MethodWithAttributesOnTypeParameters<[SomeShort] T, K>() { }
+
+    public void MethodWithAttributesOnTypeParameters<
+        [SomeLongAttribute__________________________] T,
```

Comment:
Omg I forgot this was even a thing. C# is a stupidly complicated language...

Reply:
I don't think I even realized it was a thing until I ran into it while formatting the aspnetcore repo.